### DISCUSSION TITLE
I'd like to open regular text file (text/plain; charset=us-ascii) via ENTER or RT (nothing happens)

### DISCUSSION THREAD

**Discussion by gadefox** [jarun/nnn#1726] [Q&A] [answered]
.. edit (`e` key) works fine and opens `nvim`. Any idea??? Tx

**Comment by gadefox** [comment]
Please note:
`xdg-mime query filetype ./xx` returns `text/plain`
`xdg-mime query default text/plain` returns `nvim.desktop`

and
`xdg-open ./xx` opens the text file in `nvim`

**Comment by lukasvrenner** [comment, marked as answer]
If you use the -E tag, the enter key will open text files in $EDITOR. You can store that by putting NNN_OPTS="E" in your .bash_profile or creating an alias in your .bashrc:
alias nnn='nnn -E'
